marker_api/src/ast/common/callable.rs

```diff
@@ -22,15 +22,15 @@ pub trait CallableData<'ast> {
 
     /// Returns `true`, if this callable is marked as `unsafe`.
     ///
-    /// Defaults to `false` if unspecified. Extern functions will
+    /// Defaults to `false` if unspecified. `extern` functions will
     /// also return `false` by default, even if they require `unsafe`
     /// by default.
     fn is_unsafe(&self) -> bool;
 
-    /// Returns `true`, if this callable is marked as extern. Bare functions
+    /// Returns `true`, if this callable is marked as `extern`. Bare functions
     /// only use the `extern` keyword to specify the ABI. These will currently
     /// still return `false` even if the keyword is present. In those cases,
-    /// please refer to the ABI instead.
+    /// please refer to the [`abi()`][Self::abi] instead.
```

Comment:
Do you want to adjust this to use brackets for the link target? Seeing this, I wonder if this is worth a clippy lint, that unifies all link types ^^

Reply:
Sure, I wonder if `rustfmt` can format these

Reply:
IDK, rustfmt, usually avoids doc comments, as markdown formatting can be complicated. Clippy has a few lints that parse the markdown. With that, it should be possible :)

Reply:
https://github.com/rust-lang/rust-clippy/issues/10176